testsuite/features/profiles/Makefile

```diff
@@ -6,11 +6,14 @@ TEMPLATE_KIWI_DIR   = Kiwi
 
 REGISTRY_PLACEHOLDER_NOAUTH = registry.mgr.suse.de
 REGISTRY_PLACEHOLDER_AUTH   = registry.mgr.suse.de:5000
-ENVIRONMENTS                = internal_nue internal_prv
+ENVIRONMENTS                = internal_nue internal_prv cloud_aws
 REG_REPLACEMENT_NUE_NOAUTH  = registry.mgr.suse.de
 REG_REPLACEMENT_NUE_AUTH    = registry.mgr.suse.de:5000
 REG_REPLACEMENT_PRV_NOAUTH  = registry.mgr.prv.suse.net
 REG_REPLACEMENT_PRV_AUTH    = registry.mgr.prv.suse.net:5000
+REG_REPLACEMENT_AWS_NOAUTH  = ip-172-16-1-30.eu-central-1.compute.internal
```

Comment:
I'm not super happy having this temporary AWS hostnames in our code, a better approach I think it could be to import them from environment variables loaded in .bashrc on the controller.

Reply:
In fact, both current registry.mgr.suse.de (already in bashrc) and new AWS entries could be part of .bashrc on the controller. That will keep our test code a bit more decoupled from infrastructure pointers.
But yeah, feel free to merge that first PR, and maybe we can improve later. (Unless you find time to include it in sumaform).